Bomb.py

```diff
@@ -18,20 +23,68 @@ def draw(self, win):
             file = BOMB_IMAGE_3
         elif self.timer <= FOURTH_BOMB_STATE:
             file = FIRE_IMAGE
-        else:
-            return
+
+        col = self.x // BOMB_SIZE
+        row = self.y // BOMB_SIZE
 
         img = pygame.image.load(file)
-        win.blit(img, (self.x, self.y))
+        win.blit(img, (col * BOMB_SIZE, row * BOMB_SIZE))
```

Comment:
surely this could be named better. I used `tileX`, and `tileY`, and I know it's not ideal, but I think we could look at the map in a "tile map" way, instead of "table" way